manifests/kustomize/base/cache-deployer/cluster/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: kubeflow
+
+resources:
+  - cache-deployer-clusterrole.yaml
```

Comment:
Permissions in cache-deployer-clusterrole seem too much than needed.
I believe only permission to some cluster scoped resources need to be in clusterrole, you can move other permissions to a role and use rolebinding to bind to the same sa.

That's better considering the minimal privilege principle.

Reply:
The permission limitation is revised in a different PR: https://github.com/kubeflow/pipelines/pull/3343/files#diff-2413bb0bdfa4f361948686a601a1560a. Maybe we can check in previous one first and merge the change here.

Reply:
Sure, let's merge that first. I will comment there instead.